client/src/pages/Home.js

```diff
@@ -1,8 +1,20 @@
 import PropTypes from 'prop-types';
+import styled from 'styled-components/macro';
+import { useState } from 'react';
+
+import ErrorMessage from '../components/ErrorMessage';
 import Member from '../components/Member';
 
-export default function Home({ members, onOpenModal }) {
+export default function Home({
+  members,
+  onOpenModal,
+  availableGroups,
+  deleteGroup,
+  cantDeleteGroup,
+}) {
   const orderedMembers = members.slice().sort(compareFirstName);
+  const [groupValue, setGroupValue] = useState('');
+  const [renderedGroups, setRenderedGroups] = useState(availableGroups ?? []);
```

Comment:
Setting the initial value to 'availableGroups' like you do will cause errors when you want to update the component immediately (e.g. after deleting a group on the home screen). In order to do so you need to actually update `renderedGroups` with a useEffect hook: 

`const [renderedGroups, setRenderedGroups] = useState([])

useEffect(() => {
setRenderedGroups(availableGroups)
}, [availableGroups])
`

Reply:
I realized that too (quite late unfortunately) when I implemented the Searchbar and fixed it in the next branch. Your solution however looks much more elegant, so I will see if I can include it in the searchbar branch :-).